lib/dotcom/system_status/subway.ex

```diff
@@ -6,6 +6,19 @@ defmodule Dotcom.SystemStatus.Subway do
 
   alias Alerts.Alert
 
+  @type status_time() :: :current | {:future, DateTime.t()}
+
+  @type status_entry() :: %{
+          status: atom(),
```

Comment:
This atom's`:normal` or `Alerts.Alert.effect()` right? I'd rather that be described explicitly... even better if we could have a type that's limited to the effects we care about.